app/Resources/views/base.html.twig

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>{% block title %}Profile{% endblock %}</title>
+    <title>{% block title %}OpenConext Profile{% endblock %}</title>
```

Comment:
Also, I'd like to suggest to adjust the title to `{% block title %}{% endblock %} &ndash; {{ 'profile.app.name'|trans }}` and translate the app name to SURFconext. Then override the page titles in templates extending from `base.html.twig`.